HPXMLtoOpenStudio/tests/test_hvac_sizing.rb

```diff
@@ -97,7 +97,7 @@ def test_hvac_configurations
 
           # Get more values for heat pump checks
           htg_load = autosized_bldg.hvac_plant.hdl_total
-          clg_load = autosized_bldg.hvac_plant.cdl_sens_total + autosized_bldg.hvac_plant.cdl_lat_total
```

Comment:
From doc: 
- HERS: same as ACCA except autosized heat pumps have their nominal capacity sized equal to at least the larger of heating and sensible cooling design loads.

The test here is checking capacities are equal to or greater than the larger of heating and **total** cooling design loads. Fixed it to be aligned with expectations